CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
There is a temporary developer comment `TODO: Remove before merging` which should not be left in production code. Remove this comment and remove the field if not intended to be persisted.

```suggestion
// TODO: Remove before merging
```